### DISCUSSION TITLE
Searching for collections with spaces opens the selected map instead

### DISCUSSION THREAD

**Discussion by sigidayo** [ppy/osu#27591] [Ideas] [answered]
Using a space while searching for collections opens the map instead of adding a space to the search term.
This is with default keybindings.

https://github.com/ppy/osu/assets/50233886/24f6f02e-6505-49ab-8887-011dfa7b900e

**Comment by Joehuu** [comment, marked as answer]
Already tracked in https://github.com/ppy/osu/issues/25769.
